spec/support/utilities/iis_application_pool.rb

```diff
@@ -1,15 +1,43 @@
 def has_app_pool(pool_name)
-  !(on(default, "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command \"& {Get-WebAppPoolState -Name #{@pool_name}}\"").stdout =~ /Started/i).nil?
+  command = []
+  command << "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe -noninteractive -noprofile -executionpolicy bypass -command"
```

Comment:
Running old powershell via Cygwin has an annoying bug to with STDIN redirection

https://tickets.puppetlabs.com/browse/PE-18351?focusedCommentId=394366&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-394366

Fix was implemented here;
https://github.com/puppetlabs/pe_acceptance_tests/pull/1195

Simple as adding ` < /dev/null` to the end.